src/main/java/org/springframework/data/couchbase/core/query/Query.java

```diff
@@ -104,13 +104,22 @@ public Query skip(long skip) {
 	 * Limit the number of returned documents to {@code limit}.
 	 *
 	 * @param limit
-	 * @return
+	 * @return this
 	 */
 	public Query limit(int limit) {
 		this.limit = limit;
 		return this;
 	}
 
+	/**
+	 * limit
+	 *
+	 * @return limit
```

Comment:
Please add `@since` tags to indicate when a particular class/method was introduced. This is helpful to users to know since when they can use a certain functionality.